### PR TITLE
SIDER dataset added

### PR DESCRIPTION
**Pull request by yuanqidu**

### CODE REVIEW

**Review by rbharath — COMMENTED** [review]
This looks good!

@mufeili Do you want to do a quick review pass as well before we merge in?

File: README.md [hunk 1]
```diff
@@ -47,3 +47,11 @@
...
 | 2    | GCN           | GraphConv      | 5.4312 +- 2.4279 | 2.6516 +- 0.1190 | [Mufei Li](mufeili1996@gmail.com) | [Paper](https://arxiv.org/abs/1609.02907), [Code](./examples)                            | Feb 4th, 2021  |
+
+
+### SIDER
+
+| Rank | Model         | Featurization  | Test ROC-AUC     | Validation ROC-AUC | Contact                           | References	                                                                           | Date           |
+| ---- | ------------- | -------------- | ---------------- | ------------------ | --------------------------------- | ---------------------------------------------------------------------------------------- | -------------- |
+| 1    | GCN           | GraphConv      | 0.6265 +- 0.0076 | 0.6409 +- 0.0094   | [Yuanqi Du](ydu6@gmu.edu) | [Paper](https://arxiv.org/abs/1609.02907), [Code](./examples)                            | Feb 10th, 2021 |
+| 2    | Random Forest | 1024-bit ECFP4 | 0.6504 +- 0.0011 | 0.6197 +- 0.0023   | [Yuanqi Du](ydu6@gmu.edu) | [Paper](https://www.stat.berkeley.edu/~breiman/randomforest2001.pdf), [Code](./examples) | Feb 10th, 2021 |
```
Comment (mufeili) [inline README.md:57]: For ROC-AUC, it should be the larger, the better. The numbers suggest that RF performs better in terms of the test ROC-AUC.

File: examples/gnn.py [hunk 2]
```diff
@@ -7,11 +7,12 @@
 from hyperopt import hp, fmin, tpe
 from shutil import copyfile
 
-from utils import init_trial_path, load_dataset, EarlyStopper
+import dgl
```
Comment (mufeili) [inline examples/gnn.py:10]: Not used anywhere

File: examples/gnn.py [hunk 3]
```diff
@@ -158,13 +158,14 @@ def objective(hyperparams):
       candidate_hypers,
       algo=tpe.suggest,
       max_evals=args['num_trials'])
+
   results.sort(key=lambda tup: tup[1])
+  print (results[0])
```
Comment (mufeili) [inline examples/gnn.py:163]: Why did you print this?

File: examples/gnn.py [hunk 4]
```diff
@@ -241,7 +242,7 @@ def objective(hyperparams):
...
                                      default_hyperparams)
 
-    print('Val metric for 3 runs: {:.4f} +- {:.4f}'.format(
-        np.mean(val_metrics), np.std(val_metrics)))
-    print('Test metric for 3 runs: {:.4f} +- {:.4f}'.format(
-        np.mean(test_metrics), np.std(test_metrics)))
+  print('Val metric for 3 runs: {:.4f} +- {:.4f}'.format(
+      np.mean(val_metrics), np.std(val_metrics)))
+  print('Test metric for 3 runs: {:.4f} +- {:.4f}'.format(
+      np.mean(test_metrics), np.std(test_metrics)))
```
Comment (mufeili) [inline examples/gnn.py:248]: DeepChem uses 2-space indents

**Comment by mufeili** [comment]
I left some minor comments.

**Comment by yuanqidu** [comment]
Thanks so much, mufei. Sorry about the errors. Some were left off while I was testing the code. I will correct them and PR again very soon. Happy Lunar New Year!

**Comment by yuanqidu** [comment]
I have corrected all of them, including fixing some \newline mismatch. Thanks~

**Comment by mufeili** [comment]
@yuanqidu Looks good to me. Happy Lunar New Year!

@rbharath I think this PR is ready to merge.

**Comment by rbharath** [comment]
Happy Lunar New Year as well!

Going to go ahead and merge

**Pull request merged**